docs/rate_limited_operations.rst

```diff
@@ -62,4 +62,22 @@ You can use `rate-limit` when counting items in your table:
     # Using only 15 RCU per second
     count = User.count(rate_limit = 15):
     print("Count : {}".format(count))
+    
+Unsupported Scenarios: Rate limiting Writes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Rate limiting writes are currently not supported. One workaround for this is to keep a count that gets increased based on the time passed, and only batch that number of writes (called Leaky bucket algorithm). The pseudocode below assumes writes
+are happening constantly. In the case that they're not, sleeping during low utilization could also be useful.
+
+.. code-block:: python
+
+    def write(items):
+    while True:
```

Comment:
I wrote the code in slack so I didn't get the spacing right 😬 
Can we adjust the spaces